docs/DEV-README.md

```diff
@@ -52,6 +52,15 @@ For the installation to suceed you need the following system packages (on Debian
 
 ### Run tests
 
+<<<<<<< HEAD:docs/DEV-README.md
```

Comment:
There are some artifacts here from the merge - please remove :-)

Reply:
resolved.